src/Commands/MakeModelCommand.php

```diff
@@ -67,6 +67,30 @@ protected function addSoftDelete(&$stub)
         return $this;
     }
 
+    /**
+     * Add Translatable to the given stub.
+     *
+     * @param string $stub
+     *
+     * @return $this
+     */
+    protected function addTraitTranslatable(&$stub)
```

Comment:
Can you rename the method so it's consistent with the others?

Reply:
@fletch3555 do you prefer just "addTranslatable" ?  or suggest another name?